mlflow/keras.py

```diff
@@ -60,6 +61,20 @@
 _PIP_ENV_SUBPATH = "requirements.txt"
 
 
+def _raise_deprecation_warning():
+    import keras
+
+    if Version(keras.__version__) < Version("2.3.0"):
+        warnings.warn(
+            (
+                "Support for keras < 2.3.0 has been deprecated and will be removed in a future "
+                "MLflow release"
```

Comment:
Used `future` in the case where we end up having to support keras < 2.3.0 in MLflow > 1.20.0.